File: spoj/spCANDY3.py
t = int(input())
for i in range(0,t):
    input()
    sum=0
    n = int(input())
    for _ in range(0,n):
        sum=sum+int(input())
    if sum%n==0:
        print("YES")
    else:
        print('NO')

File: spoj/spCANDY3.cpp
#include <iostream>
#define ll long long int
using namespace std;
 int main(){
     int t;
     cin>>t;
     while(t--){
         ll n;
         cin>>n;
         ll sum = 0;
         ll x;
         for(ll i=0;i<n;i++){
             cin>>x;
             sum = (sum + x)%n;
         }
         if(sum%n==0){
             cout<<"YES"<<endl;
         }
         else
         cout<<"NO"<<endl;
     }
 }